### PR TITLE
lvm: Add 'shared' parameter to lvm_lvactivate

### PR DESCRIPTION
**Pull request by vojtechtrefny**
Needed by the new blivet feature to support shared LVM setups.

### CODE REVIEW

**Review by tbzatek — APPROVED** [review]

File: src/lib/plugin_apis/lvm.api [hunk 1]
```diff
@@ -1243,6 +1243,7 @@ gboolean bd_lvm_lvrepair (const gchar *vg_name, const gchar *lv_name, const gcha
  * @vg_name: name of the VG containing the to-be-activated LV
  * @lv_name: name of the to-be-activated LV
  * @ignore_skip: whether to ignore the skip flag or not
+ * @shared: whether to activate the LV in shared mode
```
[the remark this answers is not in the document]
Comment (vojtechtrefny) [reply src/lib/plugin_apis/lvm.api]: Good point, I added more information to the docstring.

File: src/plugins/lvm-dbus.c [hunk 2]
```diff
@@ -2701,11 +2702,16 @@ gboolean bd_lvm_lvrepair (const gchar *vg_name UNUSED, const gchar *lv_name UNUS
...
  */
-gboolean bd_lvm_lvactivate (const gchar *vg_name, const gchar *lv_name, gboolean ignore_skip, const BDExtraArg **extra, GError **error) {
-    GVariant *params = g_variant_new ("(t)", (guint64) 0);
+gboolean bd_lvm_lvactivate (const gchar *vg_name, const gchar *lv_name, gboolean ignore_skip, gboolean shared, const BDExtraArg **extra, GError **error) {
+    GVariant *params = NULL;
     GVariantBuilder builder;
     GVariant *extra_params = NULL;
 
+    if (shared)
+        params = g_variant_new ("(t)", (guint64) 1 << 6);
```
[the remark this answers is not in the document]
Comment (vojtechtrefny) [reply src/plugins/lvm-dbus.c:2712]: I know, but that's internal lvmdbusd "magic constant", I don't see a better way to do it.

[the remark this answers is not in the document]
Comment (tbzatek) [reply src/plugins/lvm-dbus.c:2712]: I know, I just had to complain about that :-)

**Comment by tbzatek** [comment]
Jenkins, test this please.

**Pull request merged**